.github/workflows/gluecodium.yml

```diff
@@ -0,0 +1,207 @@
+        run: ./gradlew :lime-runtime:build
+      - name: Build and test :lime-loader
+        run: ./gradlew :lime-loader:build
+      - name: Build and test :gluecodium
+        run: ./gradlew :gluecodium:build
+      - name: Build and test :gluecodium-gradle
+        run: ./gradlew :gluecodium-gradle:build
+
+  cpp:
+    name: C++
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            !*gluecodium*
+            !*lime-*
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+      - name: Install CMake
+        uses: jwlawson/actions-setup-cmake@v1.4
+        with:
+          cmake-version: '3.15.7'
+      - name: Install ninja
+        run: sudo apt install -y ninja-build
+      - name: Install valgrind
+        run: sudo apt install -y valgrind
+      - name: Build and run functional tests
+        run: |
+          cd examples
+          ./scripts/build-cpp --valgrind --buildGluecodium
+
+  android:
+    name: Android
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            !*gluecodium*
+            !*lime-*
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+      - name: Install CMake
+        uses: jwlawson/actions-setup-cmake@v1.4
+        with:
+          cmake-version: '3.15.7'
+      - name: Install ninja
+        run: sudo apt install -y ninja-build
+      - name: Install Android SDK
+        run: |
+          ANDROID_API_LEVEL=android-28
+          ANDROID_BUILD_TOOLS_VERSION=28.0.3
+          ANDROID_CMAKE_VERSION=3.10.2.4988404
+          ANDROID_HOME=${HOME}/android-sdk-linux
+          PATH=${PATH}:${ANDROID_HOME}/tools/bin
+          wget -q "https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip" -O android-sdk-tools.zip
+          unzip -q android-sdk-tools.zip -d ${ANDROID_HOME}
+          mkdir -p ~/.android
+          touch ~/.android/repositories.cfg
+          yes | sdkmanager --licenses
+          yes | sdkmanager "platforms;${ANDROID_API_LEVEL}" > /dev/null
+          yes | sdkmanager "build-tools;${ANDROID_BUILD_TOOLS_VERSION}" > /dev/null
+          yes | sdkmanager "cmake;${ANDROID_CMAKE_VERSION}" > /dev/null
+      - name: Build and run functional tests
+        run: |
+          export ANDROID_HOME=${HOME}/android-sdk-linux
+          export ANDROID_NDK_HOME=${ANDROID_HOME}/ndk-bundle
+          export SDK_ROOT=${ANDROID_HOME}
```

Comment:
I would say that SDK_ROOT or NDK_ROOT shouldn't be used

Reply:
Will do in the follow-up commit.